zephyr/Kconfig

```diff
@@ -0,0 +1,46 @@
+if BOOTLOADER_MCUBOOT
+
+config MCUBOOT_CMAKELISTS_DIR
+	string "Path to the directory of the MCUBoot CMakeLists.txt file"
+	default "$MCUBOOT_BASE/boot/zephyr/"
+
+
+choice
```

Comment:
These options are boilerplate needed by any child image and should be put into a boilerplate file, like zephyr/subsys/logging/Kconfig.template.log_config, to be included here after setting the name to MCUBOOT or so.